feat: updated quick start (#138)
phasehq/docs: src/pages/quickstart.mdx

HTTP_PROTOCOL=https://
# WARNING: Replace these with a cryptographically strong random values. You can use `openssl rand -hex 32` to generate these.
NEXTAUTH_SECRET=82031b3760ac58352bb2d48fd9f32e9f72a0614343b669038139f18652ed1447
SECRET_KEY=92d44efc4f9a4c0556cc67d2d033d3217829c263d5ab7d1954cf4b5bfd533e58
SERVER_SECRET=9e760539415af07b22249b5878593bd4deb9b8961c7dd0570117549f2c4f32a2
# OAuth providers to enable on sign-in page (remove any that aren't required)
# Example: SSO_PROVIDERS=google,github,gitlab
SSO_PROVIDERS=google,github,gitlab
# Database credentials. Change all these values as required, except DATABASE_HOST
DATABASE_HOST=postgres # don't change this
DATABASE_PORT=5432
DATABASE_NAME=postgres-db-name
DATABASE_USER=postgres-user
DATABASE_PASSWORD=a765b221799be364c53c8a32acccf5dd90d5fc832607bdd14fccaaaa0062adfd
REDIS_HOST=redis
REDIS_PORT=6379
REDIS_PASSWORD=
NEXT_TELEMETRY_DISABLED=1
```

Run `phase secrets import .env`

```
»  phase secrets import .env
Successfully imported and encrypted 23 secrets.
To view them please run: phase secrets list
```

Once done, remove your `.env` file, as it is best practice not to leave secrets in plain text on disk:

```fish
rm .env
```

> Alternatively, you can import your secrets by dragging and dropping your `.env` file into the App's environment in the Phase Console. This may be more suitable if you want to quickly import secrets and comments denoted by `#` from your `.env` file across multiple environments.

Reviewer: ```suggestion
Alternatively, you can import your secrets by dragging and dropping your `.env` file into the App's *Secrets* tab or a specific environment in the Phase Console. This may be more suitable if you want to quickly import secrets from your `.env` file across multiple environments.
```

* Clarify that drag-and-drop import works both in *Secrets* home or a specific env
* Remove the detail about comments, was a bit confusing and the detail is unnecessary here for a quickstart imo
